roles/deploy_flask_app/files/create.yaml

```diff
@@ -0,0 +1,234 @@
+---
```

Comment:
Are these `create.yaml` and `delete.yaml` files used by the role, or are they here as examples? And do they duplicate the files in the webapp playbook's tasks?

Reply:
The files under `roles/deploy_flask_app/files` are actually copies of the files used in the playbook tasks. I'm wondering if it might be best to remove these files, as they seem to be causing more confusion than assistance.

Reply:
I might be wrong, but I do not understand why we should be needing them.

Reply:
I will remove these files in the role and point to the files in the playbooks folder in the README.